src/servers/middlewares/healthz.ts

```diff
@@ -47,3 +53,14 @@ const isRedisAvailable = async (): Promise<boolean> => {
     return false
   }
 }
+
+const isBriaAvailable = async (): Promise<boolean> => {
```

Comment:
I don't understand why bria not being healthy should cause health check to fail.
In this case bria being down would cause k8s to count the backend as being down and that would make the whole service unusable.

Instead we should be able to handle partial system degradation (eg bria down) gracefully in the api.

cc @dolcalmi

Reply:
is only mandatory where is required. for ex in main api is not https://github.com/GaloyMoney/galoy/pull/2662/files#diff-651d74d3e907cd7859cee097d1d418d4d434273dc9f3392e4faabc6446eb3974R292